temp_streaming_function.py

```
@@ -0,0 +1,394 @@
+                        persona_id = None
+                else:
+                    # Fallback to orchestrator with redirection
+                    ai_response = f"I don't recognize that persona. Available team members: {', '.join([p['id'] for p in orchestrator.scenario.get('personas', [])])}. Please use @mentions to talk to specific team members."
+                    persona_name = "ChatOrchestrator"
+                    persona_id = None
+            else:
+                # General orchestrator response - use LangChain if available
+                if orchestrator.langchain_enabled:
+                    try:
+                        # Use orchestrator's system prompt for general responses
+                        system_prompt = orchestrator.get_system_prompt()
```

Ruff (0.14.1)

Local variable `system_prompt` is assigned to but never used

Remove assignment to unused variable `system_prompt`

(F841)